internal/provisioner/helm_utils.go

```diff
@@ -266,3 +282,17 @@ func (d *helmDeployment) Version() (*model.HelmUtilityVersion, error) {
 
 	return nil, errors.Errorf("unable to get version for chart %s", d.chartDeploymentName)
 }
+
+func applyGitlabTokenIfPresent(original string) string {
+	if os.Getenv(model.GitlabOAuthTokenKey) == "" {
```

Comment:
Is there a reason we don't make this a server startup flag? There is an upcoming change to allow for all server flags to optionally be set with env vars and by setting it as a server flag we get the help text to provide context.

Reply:
No good reason not to make it a server flag, other than that usually flags require a little plumbing to pass the value around. I will add a server flag for this.

Reply:
This is potentially resolved in 7084d73. I couldn't think of any reason the server flag shouldn't just set the environment variable; let me know if you can.